src/composables/pnftInteractions.ts

```diff
@@ -1,16 +1,29 @@
-import { PNFT } from '@/common/helpers/types';
-import usePinata from '@/composables/pinata';
+import { PNFT } from '/Users/patenish/Documents/hax/helpdesk-nft-armory/src/common/helpers/types';
```

Comment:
reminder to remove the absolute path of these

Reply:
done! thanks!